posthog/settings/feature_flags.py

```diff
@@ -12,7 +12,9 @@
 default_flag_persistence = [
     # Add hard-coded feature flags for static self-hosted releases here
     "new-paths-ui-edge-weights",
-    "new-sessions-player-events-list",
+    "5730-funnel-horizontal-ui",
+    "938-breakdown-by-multiple-properties",
+    "insight-legends",
```

Comment:
CC @alexkim205

Reply:
See `5730-funnel-horizontal-ui` above too

Reply:
The "5730-funnel-horizontal-ui" is there and works but I'd hold off on releasing fully since there seems to be some value in having it side-by-side. Could we possibly A/B test this to confirm?

`insight-legends` only thing left to do is to get legend toggling working for pie charts. Not sure if this is capture anywhere so I'll cut a ticket.

Reply:
https://github.com/PostHog/posthog/issues/8236

Reply:
hell yes, worth A/B testing! Do you have a metric in mind you want to track for this? O.o

Reply:
Sounds good, feel free to create the A/B test to decide on whether we should release it or kill it.

Reply:
lmk @alexkim205 if you need any help setting that up, but hopefully the interface + docs should be sufficient to grasp things! (and always looking for feedback, let us know :D )

Reply:
<3 will try out! I think a simple metric here would be like "insight analyzed when insight=FUNNEL". I'll lyk if i run into any blockers.